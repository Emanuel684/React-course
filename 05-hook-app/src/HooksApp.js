import React from 'react'

const HooksApp = () => {
    return (
        <div>
            <h1>Hellow world!!</h1>
        </div>
    )
}

export default HooksApp;