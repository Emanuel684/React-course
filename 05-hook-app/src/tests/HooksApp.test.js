import React from 'react';
import HooksApp from '../HooksApp';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

describe('Prueba del componente HooksApp', () => {
    test('Deberia de mostrarse de manera correcta', () => {
        
        const wrapper = shallow(<HooksApp />);

        expect(wrapper).toMatchSnapshot();

    })
    
})
