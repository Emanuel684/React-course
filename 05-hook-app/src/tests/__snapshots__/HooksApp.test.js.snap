// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba del componente HooksApp Deberia de mostrarse de manera correcta 1`] = `
<div>
  <h1>
    Hellow world!!
  </h1>
</div>
`;
